pyaedt/modeler/object3dlayout.py

```diff
@@ -340,13 +485,19 @@ def enabled(self, status=True):
         bool
             `True` if succeeded.
         """
+        comp_info = self.m_Editor.GetComponentInfo(self.name)
+        for el in comp_info:
+            if "ComponentProp=" in el and "CompPropEnabled=false" in el:
+                return False
+            elif "ComponentProp=" in el and "CompPropEnabled=true" in el:
+                return True
+        return True
+
+    @enabled.setter
+    def enabled(self, status=True):
```

Comment:
```suggestion
    def enabled(self, status):
```
I think you should not put a default argument value in the setter. Maybe does't give an error, but it's never used.

Reply:
Done